src/script/event/EventRepository.ts

```diff
@@ -260,7 +258,7 @@ export class EventRepository {
     const shouldUpdatePersistedId = missedNotificationId !== notificationId;
     if (shouldUpdatePersistedId) {
       amplify.publish(WebAppEvents.CONVERSATION.MISSED_EVENTS);
-      this.notificationService.saveMissedIdToDb(this.lastNotificationId());
+      this.notificationService.saveMissedIdToDb(missedNotificationId);
```

Comment:
The value of `this.lastNotificationId()` is not being ever set, what results in always storing `undefined` under  `'z.storage.StorageKey.NOTIFICATION.MISSED'` key (amplify storage). Seems to be a leftover after these changes: https://github.com/wireapp/wire-webapp/pull/13087/files#diff-8424911ba55dc95ddd7ee35e77e651a8894faa0c33cd485cbbf62f6537fe3d24L296

Reply:
It was the reason error message was shown on every render instead of showing a message when notification's id has changed (what believe is the expected behaviour) 
<img width="1199" alt="Screenshot 2022-11-04 at 18 00 44" src="https://user-images.githubusercontent.com/45733298/200033576-87f0368a-dce2-4245-829f-b04aad50ce3a.png">